docs/dev-proc/screenshots.md

```diff
@@ -0,0 +1,54 @@
+# Taking screenshots in firmware
+
+## Introduction
+
+Dasharo EDKII fork comes with the functionality of taking screenshots. This is
+useful for creating guides, how-tos and tests. This document is meant to show
+the best practices for taking screenshots.
+
+## Resolution
+
+When taking screenshots, remember to set the display mode to 80x25. This makes
+the pictures easier to read when embedded into documentation. Notice how this:
+
+![](./images/edk2_frontpage_small.png)
+
+is easier to read than this:
+
+![](./images/edk2_frontpage_big.png)
+
+To set the mode in UEFI:
+
+- Enter the boot menu
+- Select `UEFI Shell`
+- Type in `mode 80 25` and press Enter to switch mode
+- Type in `exit` to go back to boot menu
+
+You can now enter the setup menu to take your screenshots.
+
+## Taking screenshots
+
+The key combination for taking screenshots is `LCtrl` + `LAlt` + `F12`. When
```

Comment:
Taking screenshots with PiKVM is faster in terms of copying the photos to the host PC. I would also add an info how to take screenshots when PiKVM is available.

Reply:
Added in 643d1e48a56e8bbfab2943fb3040d076413c7426